tox.ini

```diff
@@ -7,6 +7,7 @@ deps =
     py27: mock
     py27: pathlib2
     pytest-mock
+    moto[s3]
```

Comment:
The latest version of moto to support Python 2.7 is version 2.1.0. Use the latest version for Python 3.9.
This can be done similar to
```
py27: moto[s3] == 2.1.0
py39: moto[s3]
```